docs/_templates/stable_versions.html

```diff
@@ -0,0 +1,8 @@
+<div>
+    <dl>
+      <dt>Previous Releases</dt>
+      {% for version in version_list %}
```

Comment:
I think it would make sense to have newer first:
```suggestion
      {% for version in version_list | sort(reverse=True) %}
```

Probably, the right way of thing this is in `_get_version_list` with `StrictVersion` because semantic versioning:
```
from distutils.version import StrictVersion
...
    version_list.sort(key=StrictVersion)
    return version_list
```

Reply:
I think we should be ok because even with semver I can't think of a case where this will sort incorrectly just using a string sort of the version number. The only version tag I could see having a potential issue is if we did prereleases where `0.2.0` would be less than `0.2.0rc0`. But for the metapackage we don't do pre-releases this won't come up.

But I think pushing a follow up to use [packaging](https://packaging.pypa.io/en/latest/index.html)  (which replaced the deprecated `distutils.version` module) is a good idea.